website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting started as a TAG Environmental Sustainability Contributor
+description: Getting started as a TAG Environmental Sustainability Contributor
+slug: tbd
+weight: 1
+date:   2023-11-07 00:00:00 +0000
+author: Antonio Di Turi, Atreay Kukanur, Mfon Nta
+reviewers: Kristina Devochko, Leonard Pahlke, Marlow Weston
+---
+
+## Getting started as a TAG Environmental Sustainability Contributor
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-11-getting-started-as-a-tag-environmental-sustainability-contributor/tag-env-logo.png" alt="TAG Environmental Sustainability Logo"></p>
+
+If you are reading this blog it might be that you are looking forward to starting contributing to sustainability in the cloud native world.  This document will help you, and you can start by becoming a member of the TAG or one of its associated working groups.
+
+Starting to contribute to open source and cloud native communities may often seem challenging and scary for beginners, but there are many ways you can embark on this journey. One way to join the community is to become a member of one of the Cloud Native Computing Foundation Technical Advisory Groups (CNCF TAG).
+
+The Technical Advisory Group consists of members that focus on a specific area of the CNCF landscape by providing guidance, addressing challenges from the community and establishing domain-specific standards and best practices.
+
+CNCF Technical Advisory Group Environmental Sustainability is always open for new members and we hope that this article can help you clarify any concerns and questions you may have around joining the TAG, as well as make it easier for you to take those first steps in joining the community.
+
+With this blog post we would like to cover a few topics that you may be wondering about when it comes to getting started as a TAG ENV Contributor:
```

Comment:
```suggestion
If you are curious about the TAG and play with the ideas to contribute you likely ask yourself the following questions.
 ```

Reply:
@AntonioDiTuri this one doesn't seem to be resolved?